packages/prediction/src/type.ts

```diff
@@ -38,6 +38,14 @@ export interface PredictionConfig {
   displayedDecimals: number
   token: Token
   tokenBackgroundColor: string // For selector svg token for prediction page.
+
+  isAIPrediction?: boolean // AI-based prediction market
+
+  // Decimals to accommodate varying price sources
+  AIPriceDecimals?: number
+  lockPriceDecimals?: number
+  closePriceDecimals?: number
+  livePriceDecimals?: number
```

Comment:
If these are ai specific, can we group them together?
```typescript
type AIPredictionConfig = {}

type PredicitonConfig = {
  ai?: PredictionConfig
}
```

Reply:
I am thinking that in future if we move to other price sources for Normal prediction, these can be useful in general? Except `AIPriceDecimals`. @chefjackson

Reply:
sure. We can keep those generic attributes. How about
```typescript
type AIPredictionConfig = {
  aiPriceDecimals?: number
}

type PredicitonConfig = {
  ai?: PredictionConfig
}
```

In this way we can remove `isAIPrediction` and simply check if `ai` is specified.

Reply:
and let's use lower case for the first letter

Reply:
Sure